macros/models/_sector/add_tx_from_and_to.sql

```diff
@@ -0,0 +1,18 @@
+{% macro add_tx_from_and_to(
+    model_cte
+    ,blockchain
+    )
+%}
+
+select
+    model.*
+    , tx."from" as tx_from
+    , tx."to" as tx_to
+from {{model_cte}} model
+inner join {{source(blockchain, 'transactions')}} tx
+    on model.block_number = tx.block_number
+    and model.tx_hash = tx.hash
+    {% if is_incremental() %}
+    and tx.block_time > date_trunc('day', now() - interval '1' day)
```

Comment:
we can universally simplify incremental logic to:
`WHERE {{incremental_predicate('tx.block_time')}}`

PR here:
https://github.com/duneanalytics/spellbook/pull/4487/files#diff-dc8d4490c16a624e93e14dcb8cae5c6f8f53697c962e2e4fae095432d705eb4c